charts/prometheus-cloudwatch-exporter/values.yaml

```
@@ -34,6 +34,9 @@ service:
   annotations: {}
   labels: {}
 
+deployment:
+  labels: {}
```

I suggest to either use:
```suggestion
deployment:
  podLabels: {}
```
or
```suggestion
pod:
  labels: {}
```